pills/17-nixpkgs-overriding-packages.xml

```diff
@@ -44,31 +44,20 @@
   <section>
     <title>Fixed point</title>
     <para>
-      The fixed point with lazy evaluation is crippling but about necessary in a language like Nix. It lets us achieve something similar to what we'd do imperatively.
+      A fixed point is a common pattern in lazy languages to ensure that functions that would otherwise return non fully evaluated values do return fully evaluated values. Let's unpack this idea. Remember that in Nix functions are lazy. Consider:
     </para>
+    <screen><xi:include href="./17/laziness.txt" parse="text" /></screen>
```

Comment:
By this point, people should know what laziness means:

https://github.com/NixOS/nix-pills/blob/a66fd074cf4b69ea7105ab8dea28903ecc7a8447/pills/04-basics-of-language.xml#L338-L345